tla/consensus/abs.tla

```diff
@@ -0,0 +1,58 @@
+---- MODULE abs ----
+\* Abstract specification for a distributed consensus algorithm.
+\* Assumes that any node can inspect the state of all other nodes. 
```

Comment:
Perhaps, reword to something like "can atomically inspect the state of all other nodes"?

Reply:
https://github.com/microsoft/CCF/pull/6438/commits/1d131f6eeb43042f48cada238fe7474957f843ac